docs/conversation.md

```diff
@@ -425,3 +426,21 @@ Get all (for moderators and in case of "free selection") or the assigned breakou
         + `400 Bad Request` When the conversation is a breakout room
         + `403 Forbidden` When the current user is not a moderator/owner or the conversation is not a public conversation
         + `404 Not Found` When the conversation could not be found for the participant
+
+## change the permission to mention everyone using `@all`
+* Required capability: `can-mention-everyone`
+* Method: `PUT`
+* Endpoint: `/room/{token}/can-mention-everyone`
+* Data:
+
+| field        | type | Description                                                               |
+|--------------|------|---------------------------------------------------------------------------|
+| `permission` | int  | See [Constants - Can mention everyone](constants.md#can-mention-everyone) |
```

Comment:
```suggestion
| `config` | int  | See [Constants - Can mention everyone](constants.md#can-mention-everyone) |
```

permission sounds weird, since it's more targeting a group